<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorSignature</class>
 <widget class="QDialog" name="EditorSignature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="label_tel_2">
     <property name="text">
      <string>Телефон</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="6">
    <widget class="QLineEdit" name="signatory_tel_3"/>
   </item>
   <item row="7" column="0" rowspan="2">
    <widget class="QLabel" name="label_post_3">
     <property name="text">
      <string>Должность третьего подписанта</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QLineEdit" name="signatory_tel_2"/>
   </item>
   <item row="7" column="3">
    <widget class="QLineEdit" name="signatory_fio_3"/>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="signatory_fio_2"/>
   </item>
   <item row="7" column="5">
    <widget class="QLabel" name="label_tel_3">
     <property name="text">
      <string>Телефон</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_fio_2">
     <property name="text">
      <string>Инициалы, Фамилия</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1" rowspan="2">
    <widget class="QTextEdit" name="signatory_post_3"/>
   </item>
   <item row="5" column="1" rowspan="2">
    <widget class="QTextEdit" name="signatory_post_2"/>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QLabel" name="label_post_1">
     <property name="text">
      <string>Должность первого подписанта</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_fio_3">
     <property name="text">
      <string>Инициалы, Фамилия</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="4">
    <widget class="QTextEdit" name="signatory_post_1"/>
   </item>
   <item row="5" column="0" rowspan="2">
    <widget class="QLabel" name="label_post_2">
     <property name="text">
      <string>Должность второго подписанта</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="signatory_tel_1"/>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_tel_1">
     <property name="text">
      <string>Телефон</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_fio_1">
     <property name="text">
      <string>Инициалы, Фамилия</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="signatory_fio_1"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditorSignature</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditorSignature</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
